assemble/src/main/scripts/create-jshell.sh

```diff
@@ -36,6 +36,25 @@ function addAccumuloAPI() {
   echo
 }
 
+function addClientBuild() {
+  echo "URL clientPropUrl = 
+    AccumuloClient.class.getClassLoader().getResource(\"accumulo-client.properties\");
+  String accumuloProp; AccumuloClient client = null;
+  
+  // Does Accumulo properties exists?
+  if (clientPropUrl != null) {
+  
+    // Build Accumulo Client
+    accumuloProp = clientPropUrl.getFile();
+    System.out.println(\"Building Accumulo client using properties file below: \\n\"+accumuloProp);
+    client = Accumulo.newClient().from(accumuloProp).build();
+    System.out.println(\"Use \"+'\"'+\"client\"+'\"'+\" to interact with Accumulo \\n\");
+  } 
+  if (clientPropUrl == null)
```

Comment:
could this be an else on the prev if stmt?

Reply:
I tried using an else but I kept getting "Java reached end of file while parsing" error while JShell attempted to execute the startup script. Which is why I included a separate if statement.

Reply:
That's weird. I wonder if you had a typo with curly braces. It might be easier to see typos if you use a "heredoc" to print the java code instead of `echo`. That way you don't have as much quoting and escaping, and it's easier to read it as java code. See my comment above.

Reply:
`  if (clientPropUrl != null) {
  
    // Build Accumulo Client
    accumuloProp = clientPropUrl.getFile();
    System.out.println("Building Accumulo client using properties file below: \n" + accumuloProp);
    client = Accumulo.newClient().from(accumuloProp).build();
    System.out.println("Use " + "\"client\"" + " to interact with Accumulo \n");
  } else {
      System.out.println("accumulo-client.properties was not found on the classpath \n");
  }`
I don't believe it's a typo but rather a weird JShell behavior. This morning I placed the else statement in a different position (See above) and it did not throw an error when booting up JShell. I tried looking into the JShell documentation to find out why this occurred but couldn't find anything. If I had to guess JShell may use a specific control flow structure to help identify the start and end of an if/else statement